app/assets/stylesheets/components.scss

```diff
@@ -115,15 +115,52 @@
 }
 
 .circuit-card-name {
-  display: inline-block;
   float: left;
   font-weight: 600;
-  margin-top: 10px;
-  max-height: 30px;
-  max-width: 150px;
-  overflow: hidden;
+  margin-top: 5px;
+  position: relative;
   text-align: left;
-  text-overflow: ellipsis;
+
+  p {
+    max-height: 50px;
+    max-width: 150px;
+    overflow: hidden;
+  }
+
+  .tooltiptext {
+    background-color: var(--green);
+    border-radius: 6px;
+    bottom: 100%;
+    color: var(--white);
+    left: 50%;
+    margin-left: -60px;
+    opacity: 0;
+    padding: 5px 0;
+    position: absolute;
+    text-align: center;
+    transition: opacity .5s;
+    visibility: hidden;
+    width: 120px;
```

Comment:
Use this rather, max-width: 160px

Reply:
thanks for pointing it , smaller circuit names are perfectly fitting without any extra space on its left and right that is looking really good!!
changes made accordingly!!